### PR TITLE
Add print stylesheet

### PR DESCRIPTION
**Pull request by cailyncodes**
Simplify implementation for User story #3. This PR adds a print style sheet with the use of CSS media queries.

This implementation has a known limitation on being able to render a display on Firefox in the browser. This is due to how Firefox implements printing (versus Chrome-based browsers). For an implementation that provides a potential solution, see #34 (on branch 3-print-roster-with-context).

### CODE REVIEW

**Comment by cailyncodes** [comment]
@cmgiven Optional review

File: ClientApp/src/containers/App/App.test.tsx [hunk 1]
```diff
@@ -16,3 +17,19 @@ it('renders without crashing', () => {
...
+		<MockedProvider>
+			<BrowserRouter>
+				<div className="print">
+					<App />
+				</div>
+			</BrowserRouter>
+		</MockedProvider>
+	);
+
+	expect(wrapper).toMatchSnapshot();
```
Comment (cmgiven) [inline ClientApp/src/containers/App/App.test.tsx]: Seems like this wouldn't actually test anything other than that the className has been added. The snapshots just run a diff on the markup; they're not doing a visual comparison.

I'm not sure that there is a way to test this feature without adding visual regression testing.

Comment (cailyncodes) [reply ClientApp/src/containers/App/App.test.tsx]: Welp okay yeah I'm not sure we can test this then. I'll clean up the test and style sheet.

**Comment by cailyncodes** [comment]
Closes #3

**Pull request merged**